src/webhooks/handleCheckoutSessionCompleted.ts

```diff
@@ -59,6 +59,17 @@ export default async function handleCheckoutSessionCompleted(
     throw err;
   }
 
+  try {
+    await updateUserTier(user.uuid, price.product as string, config);
+  } catch (err) {
+    log.error(
+      `Error while updating user tier: email: ${session.customer_email}, planId: ${price.product} `,
+    );
+    log.error(err);
```

Comment:
can't say for sure but more often than not logging systems logs `[object Object]` unless you 
```tsx
JSON.stringify(err,null,'\t')
```